api/routers/cameras.py

```diff
@@ -297,3 +299,45 @@ async def enable_video_live_feed(camera_id: str, disable_other_cameras: Optional
     config_dict[f"Source_{index}"]["LiveFeedEnabled"] = "True"
     success = update_config(config_dict, True)
     return handle_response(None, success, status.HTTP_204_NO_CONTENT)
+
+
+@cameras_router.get("/{camera_id}/roi_contour")
+async def get_roi_contour(camera_id: str):
+    """
+        Get the contour of the RoI
+    """
+    roi_file_path = ObjectsFilteringPostProcessor.get_roi_file_path(camera_id, settings.config)
+    roi_contour = ObjectsFilteringPostProcessor.get_roi_contour(roi_file_path)
+    if roi_contour is None:
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail=f"There is no defined RoI for {camera_id}")
+    return roi_contour.tolist()
+
+
+@cameras_router.put("/{camera_id}/roi_contour", status_code=status.HTTP_201_CREATED)
+async def add_or_replace_roi_contour(camera_id: str, body: ContourRoI, reboot_processor: Optional[bool] = True):
+    """
+        Define a RoI for a camera or replace its current one.
+        A RoI is defined by a vector of [x,y] duples, that map to coordinates in the image.
+    """
+    roi_file_path = ObjectsFilteringPostProcessor.get_roi_file_path(camera_id, settings.config)
+    dir_path = Path(roi_file_path).parents[0]
+    Path(dir_path).mkdir(parents=True, exist_ok=True)
+    roi_contour = np.array(body.contour_roi, dtype=int)
+    Path("/my/directory").mkdir(parents=True, exist_ok=True)
+    np.savetxt(roi_file_path, roi_contour, delimiter=',', fmt='%i')
+    restart_processor() if reboot_processor else True
```

Comment:
This change only affects one postprocessing step so restarting the processor seems overkill. Maybe we need to implement something to just update part of the processor. (Not for this PR)

Reply:
Yes, it would also be great for enabling the video logger, definitely a ticket to add to the icebox

Reply:
I created the ticket